packages/saltcorn-data/db/state.ts

```diff
@@ -593,6 +597,9 @@ class State {
     withCfg("headers", []).forEach((h: any) => {
       if (!this.headers[name].includes(h)) this.headers[name].push(h);
     });
+    const routes = withCfg("routes", []);
```

Comment:
Am I right in thinking that this way only one plugin can supply routes? If  possible I think following the same pattern with forEach followed by push would allow multiple plugins to supply routes.

Reply:
Hi, the routes are added to the 'plugin_routes' member, which maps plugin names to routes of a plugin. So multiple plugins are possible. 

But when one plugin with routes gets reloaded, then all routes of all plugins are remounted into the server, but I think that's just an optimization thing.

Reply:
yes i see